p2p/peerResend.go

```diff
@@ -0,0 +1,99 @@
+package p2p
+
+import (
+	"crypto/sha1"
+	"sync"
+	"time"
+)
+
+type PeerResend struct {
+	mtx      sync.RWMutex
+	buckets  []*PRBucket
+	pc       int
+	interval time.Duration
+	stopper  chan interface{}
+}
+
+type PRBucket struct {
```

Comment:
what does PR stands for? And does this struct needs to be exposed (capitalized)?

Reply:
It used to stand for "peer resend" in this case but it should not be exported.